config/sync/filter.format.stanford_html.yml

```diff
@@ -58,7 +58,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<br> <b> <em> <strong> <cite> <blockquote cite> <pre class> <code data-* class> <ul type class id> <ol start type class id> <li class id> <dl class> <dt> <dd class> <h2 id class> <h3 id class> <h4 id class> <h5 id class> <div class id> <table class id> <caption> <tbody class> <thead> <tfoot> <th scope class id> <td colspan rowspan class id> <tr id> <sup> <sub> <i class> <hr> <s> <aside class> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt data-view-mode title> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title name class> <p class id> <img class src alt title>'
+      allowed_html: '<br> <b> <em> <strong> <cite> <blockquote cite> <pre class=""> <code data-* class=""> <ul type id class=""> <ol start type id class=""> <li id class=""> <dl class=""> <dt> <dd class=""> <h2 id class=""> <h3 id class=""> <h4 id class=""> <h5 id class=""> <div id class=""> <table id class=""> <caption> <tbody class=""> <thead> <tfoot> <th scope id class=""> <td colspan rowspan id class=""> <tr id> <sup> <sub> <i class=""> <hr> <s> <aside class=""> <drupal-media data-entity-type data-entity-uuid data-align data-caption data-* alt data-view-mode title> <img src alt title class=""> <a href hreflang data-entity-substitution data-entity-type data-entity-uuid title name class=" su-button su-button--big su-button--secondary su-link--action"> <p id class=" plain-text su-intro-text su-font-splash su-quote-text su-drop-cap su-related-text su-callout-text su-subheading"> <abbr title="">'
```

Comment:
`class=""` is bad. I know this is generated from the UI but it should be fixed.

Reply:
fixed.